develop/dev-guide-aws-quicksight-integration.md

```diff
@@ -0,0 +1,86 @@
+---
+title: QuickSight Integration Guide
+summary: Learn how to integrate TiDB Cloud Dedicated Tier with Amazon QuickSight.
+---
+
+# Integrate TiDB with Amazon QuickSight
+
+This document provides a high-level introduction to Amazon QuickSight and describes how to integrate Amazon QuickSight with TiDB.
+
+If you are interested in learning more about TiDB and ProxySQL, you can find some useful links as follows:
+
+- [TiDB Cloud](https://docs.pingcap.com/tidbcloud)
+- [TiDB Developer Guide](/develop/dev-guide-overview.md)
+- [Amazon QuickSight Documentation](https://docs.aws.amazon.com/quicksight/latest/user/welcome.html)
+
+## What is ProxySQL?
+
+[Amazon QuickSight](https://aws.amazon.com/quicksight/) is a cloud-scale business intelligence (BI) service that you can use to deliver easy-to-understand insights to the people who you work with, wherever they are. Amazon QuickSight connects to your data in the cloud and combines data from many different sources. In a single data dashboard, QuickSight can include AWS data, third-party data, big data, spreadsheet data, SaaS data, B2B data, and more. As a fully managed cloud-based service, Amazon QuickSight provides enterprise-grade security, global availability, and built-in redundancy. It also provides the user-management tools that you need to scale from 10 users to 10,000, all with no infrastructure to deploy or manage.
+
+## Why Amazon QuickSight integration?
+
+Every day, the people in your organization make decisions that affect your business. When they have the right information at the right time, they can make the choices that move your company in the right direction.
+
+Here are some of the benefits of using Amazon QuickSight for analytics, data visualization, and reporting:
+
+- The in-memory engine, called SPICE, responds with blazing speed.
+- No upfront costs for licenses and a low total cost of ownership (TCO).
+- Collaborative analytics with no need to install an application.
+- Combine a variety of data into one analysis.
+- Publish and share your analysis as a dashboard.
+- Control features available in a dashboard.
+- No need to manage granular database permissions—dashboard viewers can see only what you share.
+
+## Create TiDB Cluster
```

Comment:
One suggestion I have is to consider adding a new section called "Prerequisites" here. In this section, you could list out the necessary requirements for following the instructions in the rest of the document, such as a TiDB cluster, an Amazon QuickSight account, and ensuring that Amazon QuickSight can connect to your TiDB cluster. This will help ensure that readers have everything they need to get started and avoid any potential roadblocks.

Reply:
Got it. Changed.